packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -319,7 +334,12 @@ export const Webchat = forwardRef((props, ref) => {
     )
     if (!Array.isArray(blockInputs)) return false
     for (const rule of blockInputs) {
-      if (rule.match.some(regex => regex.test(input.data))) {
+      if (
+        rule.match.some(regex => {
```

Comment:
let's try yo reduce complexity to the minimum of every single function. We sholdn't wait until they are much too complex, we want it always easy :-)
What about creating a getBlockInputs() function which deserializes the regex?